src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -43,7 +60,7 @@ const BasicNetworkFields: React.FC<BasicNetworkFieldsProps> = ({ hostSubnets })
       <InputField
         label="API Virtual IP"
         name="apiVip"
-        helperText="Virtual IP used to reach the OpenShift cluster API. Make sure that the VIP's are unique and not used by any other device on your network."
+        helperText={`Virtual IP used to reach the OpenShift cluster API. Make sure that the VIP's are unique and not used by any other device on your network${subnet}.`}
```

Comment:
miss
```suggestion
        helperText={`Virtual IP used to reach the OpenShift cluster API. Make sure that the VIP's are unique and not used by any other device on your network ${subnet}.`}
```
missing space ^

Reply:
No, that was intentional :-) Otherwise the `dot` would be after a space, see how `subnet` is defined above.

Reply:
Ah, I see, thanks :)